src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -569,6 +570,16 @@ public void selectQuestionNumber(int qnNumber, int newQnNumber) {
         selectDropdownByVisibleValue(qnNumSelect, String.valueOf(newQnNumber));
     }
 
+    public boolean isSelectQuestionNumberEnabled(int qnNumber) {
+        WebElement qnNumSelect = browser.driver.findElement(By.id("questionnum-" + qnNumber));
```

Comment:
Yes, but `Select` objects don't have any methods to check if the element is enabled or not. So, can't use it in `isSelectQuestionNumberEnabled()`.